gui/components/wallet/send_popup.py

```diff
@@ -0,0 +1,59 @@
+import time
+
+import allure
+
+import configs
+import constants
+import driver
+from driver.objects_access import wait_for_template
+from gui.components.base_popup import BasePopup
+from gui.components.community.authenticate_popup import AuthenticatePopup
+from gui.elements.button import Button
+from gui.elements.object import QObject
+from gui.elements.text_edit import TextEdit
+from gui.elements.text_label import TextLabel
+
+
+class SendPopup(BasePopup):
+
+    def __init__(self):
+        super().__init__()
+        self._tab_item_template = QObject('tab_Status_template')
+        self._search_field = TextEdit('edit_TextEdit')
+        self._token_list_item = QObject('o_TokenBalancePerChainDelegate_template')
+        self._amount_text_edit = TextEdit('amountInput_TextEdit')
+        self._paste_button = Button('paste_StatusButton')
+        self._enf_address_text_edit = TextEdit('ens_or_address_TextEdit')
+        self._my_accounts_tab = Button('accountSelectionTabBar_My_Accounts_StatusTabButton')
+        self._account_list_item = QObject('status_account_WalletAccountListItem_template')
+        self._arbitrum_network = QObject('arbitrum_StatusListItem')
```

Comment:
i dont get where is this arbitrum definition is coming from. Normally on Simple send is suggested to be send from the fastest network possible, so it could be any of L2 networks. In my case i have Optimism as suggestion for example now. So the test would fail if Optimism will be suggested then? 

<img width="1312" alt="Screenshot 2023-11-24 at 10 16 05" src="https://github.com/status-im/desktop-qa-automation/assets/82375995/59a377b0-a1f7-497c-9688-51050be3cd67">

Reply:
I user data I saved tokens from arbitrum:
![image](https://github.com/status-im/desktop-qa-automation/assets/128374224/d38e0589-4922-4d6a-8f50-5a500985bf8b)